src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import { getNumbers } from '../../utils';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const lastItemOnPage = Math.ceil(total / perPage);
+  const pageLinks = getNumbers(1, lastItemOnPage);
+
+  function prevPage(page: number) {
+    if (page > 1) {
+      onPageChange(page - 1);
+    }
+  }
+
+  function nextPage(page: number) {
+    if (page < lastItemOnPage) {
+      onPageChange(page + 1);
+    }
+  }
+
```

Comment:
use arrow functions + 
```suggestion
  const prevPage = () => {
    if (currentPage > 1) {
      onPageChange(page - 1);
    }
  }

  const nextPage = () =>  {
    if (currentPage < lastItemOnPage) {
      onPageChange(page + 1);
    }
  }

```